content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/title.html

```diff
@@ -20,9 +20,8 @@
      data-cmp-data-layer="${title.data.json}"
      id="${component.id}"
      class="cmp-title">
-    <h1 class="cmp-title__text" data-sly-element="${title.type}">
-        <a data-sly-unwrap="${!title.linkURL || title.linkDisabled}" class="cmp-title__link" href="${title.linkURL}"
-           data-cmp-clickable="${title.data ? true : false}">${text}</a>
-    </h1>
+    <h1 class="cmp-title__text" data-sly-element="${title.type}"><a data-sly-unwrap="${!title.linkURL || title.linkDisabled}"
```

Comment:
please add a new line before the <a> element

Reply:
When a new line is added and the link element get removed the text of the title is getting some ugly whitespace and the ui test is failing

Reply:
I'm not fond of this change either but @bpauli has a point, in case the `a` element unwraps, your get a:
```
<h1>
    Title text
</h1>
```
which renders extra whitespace before the title text.

I would, at least try to break the `a` element in a more readable way:
```
<h1 class="cmp-title__text" data-sly-element="${title.type}"><a 
    data-sly-unwrap="${!title.linkURL || title.linkDisabled}"
```

Reply:
I see, thx. 
+1 for Vlad's suggestion.